app/views/podcasts/show.rss.builder

```diff
@@ -115,13 +115,20 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
           xml.itunes :keywords, ep.keywords.join(',') unless ep.keywords.blank?
           xml.itunes(:isClosedCaptioned, 'Yes') if ep.is_closed_captioned
 
-          xml.media(:content,
-            fileSize: ep.file_size,
-            type: ep.content_type,
-            url: ep.media_url) if ep.media?
-
           xml.content(:encoded) { xml.cdata!(ep.content) } unless ep.content.blank?
         end
+
+        if ep.media?
+          if @podcast.restrictions.present?
+            xml.media(:content, fileSize: ep.file_size, type: ep.content_type, url: ep.media_url) do
+              @podcast.restrictions.each do |r|
```

Comment:
I think this should just be in the item, not in the `media:content` block?

the examples in the mrss spec and the spotify docs show `media:restriction` as a child tag of item, not contained in a `media:content` block:
https://podcasters.spotify.com/terms/Spotify_Podcast_Delivery_Specification_v1.6.pdf
https://www.rssboard.org/media-rss#media-restriction

Reply:
Oh, interesting.  In the spec:

> The following elements are optional and may appear as sub-elements of <channel>, <item>, <media:content> and/or <media:group>.

Would be convenient if we could just set it at the `channel` level.  But sounds like Spotify wants it at the `item`, so I guess that's what I should do!